client/src/index.jsx

```diff
@@ -1,15 +1,37 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import $ from 'jquery';
+import Bonuses from './components/bonuses.jsx';
+
 
 class App extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      bundleNumber: 1, //number
+      bonus: null, //object
+      selectedSong: '', //url
+    };
+
+    this.selectSong = this.selectSong.bind(this);
+  }
+  componentDidMount() {
+    $.get(`/bonus/${this.state.bundleNumber}`, (data) => {
+      this.setState({ bonus: data });
+    });
```

Comment:
Consider handling errorful returned data somehow here. If you get a misshapen or non- object back for your `this.state.bonus`, you'll be passing through a faulty prop to your render of the Bonuses component.

Reply:
I will keep that in mind as I move forward.